### PR TITLE
OpenMP Fix on M1

### PR DESCRIPTION
**Pull request by dcoeurjo**
# PR Description

*your description here*

# Checklist

- [x] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [x] No warning raised in Debug mode.
- [x] All continuous integration tests pass (Github Actions & appveyor)

### CODE REVIEW

File: cmake/DGtalConfig.cmake.in [hunk 1]
```diff
@@ -86,20 +86,33 @@ if (@QT5_FOUND_DGTAL@) #if QT5_FOUND_DGTAL
 endif (@QT5_FOUND_DGTAL@)
 
 if(@OPENMP_FOUND_DGTAL@) #if OPENMP_FOUND_DGTAL
-  if(APPLE)
-    message("-- Setting OpenMP flags on MacOs. Assuming `brew install libomp`")
-    if(CMAKE_C_COMPILER_ID MATCHES "Clang\$")
+if(APPLE)
```
Comment (kerautret) [inline cmake/DGtalConfig.cmake.in:89]: indent as before change ?

**Review by kerautret — APPROVED** [review]
small perhaps indent but fine for me, I am testing on my Mac (not on M1 ;))

**Comment by kerautret** [comment]
small perhaps indent but fine for me, I am testing on my Mac (not on M1 ;))

**Comment by kerautret** [comment]
Else looks fine on my Intel Mac 👍🏻

**Comment by dcoeurjo** [comment]
Thx. Updated

**Pull request merged**